upstart_msh/cob.conf

```diff
@@ -1 +1,39 @@
-../upstart/cob.conf
\ No newline at end of file
+#
+# Author: Mike Purvis <mpurvis@clearpathrobotics.com>
```

Comment:
where does this header come from? we should rather use the ones from https://github.com/ipa320/setup/tree/master/templates